test/cpp/qps/scenario_runner.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Local QPS benchmark runner for the OSS Benchmark loadtest configurations.
+
+This tool will run a scenario locally, either already extracted from
+scenario_config_extractor, or extracted from a benchmark loadtest config. The
```

Comment:
Nit: Do you mean `scenario_config_exporter`?

Reply:
Fixed, thanks.